nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/main/java/org/apache/nifi/processors/kafka/GetKafka.java

```diff
@@ -184,6 +195,9 @@
     private volatile long deadlockTimeout;
 
     private volatile ExecutorService executor;
+    private String zookeeperConnectionString;
```

Comment:
These three will be accessed by different threads and should be marked volatile.